client/src/components/BuddyRequestFormular.js

```diff
@@ -0,0 +1,83 @@
+import styled from "@emotion/styled";
```

Comment:
The name of the component is a mix of German and English. Maybe it can be named `BuddyRequestForm`. The exported component is named `ConcertCardBig`. If there is only one export in this file, maybe the file can be named like the standard export (in this case `ConcertCardBig`).

Reply:
Oops.. :)